packages/react-instantsearch-hooks-dom/src/widgets/CurrentRefinements.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { useCurrentRefinements } from 'react-instantsearch-hooks';
+
+import { CurrentRefinements as CurrentRefinementsUiComponent } from '../ui/CurrentRefinements';
+
+import { isModifierClick } from './lib/isModifierClick';
+
+import type { CurrentRefinementsProps as CurrentRefinementsUiComponentProps } from '../ui/CurrentRefinements';
+import type { UseCurrentRefinementsProps } from 'react-instantsearch-hooks';
+
+export type CurrentRefinementsProps = Omit<
+  CurrentRefinementsUiComponentProps,
+  'disabled' | 'onClick'
```

Comment:
I don't think we should expose `items` and `hasRefinements` from the UI props, if I'm not mistaken.

```suggestion
  'disabled' | 'onClick' | 'items' | 'hasRefinements'
```

Reply:
Good catch!